app/models/merchant.rb

```diff
@@ -21,4 +21,16 @@ def self.group_by_enabled
   def self.group_by_disabled
     Merchant.where('status = ?', false)
   end
+
+  def self.top_five_by_revenue #is this working?? transactions with at least one success??
```

Comment:
Let's double check the testing on the model side as well to determine we're testing for edge cases.